onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -399,6 +414,52 @@ def _get_quantized_weight(self, initializer, qType):
 
         return weight
 
+    def _get_quantized_weight_per_channel(self, initializer, qType, channel_axis, symmetric=False, reduce_range=False):
```

Comment:
>_get_quantized_weight_per_channel [](start = 8, length = 33)

It looks very similar as quantize_weight_per_channel. Could you please combine them

Reply:
I agree. I will combine them together. Thanks!